README.md

```diff
@@ -16,14 +10,79 @@ Kubernetes stack cookbook to work with Kubernetes
 
   ```bash
   $ cd ~/teracy-dev/workspace
-  $ git checkout <your_forked_repo>
+  $ git clone git@github.com:<your_forked_repo>/kubernetes-stack-cookbook.git
   $ cd kubernetes-stack-cookbook
   $ git remote add upstream git@github.com:teracyhq-incubator/kubernetes-stack-cookbook.git
   ```
 
-- `$ vagrant reload --provision` to update the dev-setup from this project into the teracy-dev's VM.
+- remember to run `$ vagrant reload --provision` to update the dev-setup from this project into the teracy-dev's VM.
   After that, you should be ready to work on the project.
 
+## How to use
```

Comment:
@hoatle standard Chef cookbook ? it's a standalone chef cookbook( run without teracy-dev)
 or run with teracy-dev